<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>
  <env name="ROS_LOG_DIR" value="/tmp/"/>

  <node pkg="tf" type="static_transform_publisher" name="wheel2_translation" args=" 0.23625  -0.13  0.0575  0 0 0 /base_link /wheel2_base 50"/>
  
  <node pkg="wheel_bridge" type="wheel_bridge" name="wheel2" output="screen" respawn="true">
    <param name="serial_dev" value="/dev/serial/by-id/usb-Teensyduino_USB_Serial_5952630-if00" />
    <param name="baudrate" type="int" value="115200"/>

    <param name="input_topic" value="/wheel2" />
    <param name="output_topic" value="/wheel2_meta" />

    <param name="tf2_base_link" value="wheel2_base" />
    <param name="tf2_output" value="wheel2" />
  </node>

</launch>
